src/messaging/messages/message.js

```diff
@@ -8,7 +8,7 @@ export default class Message {
   /**
    * @param {object} body - A plain JS object (with no methods) representing a body of the message.
    */
-  constructor (body) {
+  constructor (body = {}) {
```

Comment:
Should the documentation for this constructor be changed to note that `body` is optional and has a default value?

Reply:
This is a good catch! I will update description to reflect that.